public/video-ui/src/components/Pluto/PlutoProjectPicker.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _set from 'lodash/fp/set';
+import { ManagedForm, ManagedField } from '../ManagedForm';
+import SelectBox from '../FormFields/SelectBox';
+
+class PlutoProjectPicker extends React.Component {
+  static propTypes = {
+    video: PropTypes.object.isRequired,
+    saveVideo: PropTypes.func.isRequired
+  };
+
+  hasPlutoCommissions = () => this.props.pluto && this.props.pluto.commissions.length !== 0;
+
+  componentWillMount() {
+    if (!this.hasPlutoCommissions()) {
+      this.props.plutoActions.getCommissions().then(() => {
+        const {plutoData} = this.props.video;
+        if (plutoData.commissionId) {
+          this.props.plutoActions.getProjects(plutoData);
+        }
+      });
+    }
+  }
+
+  onCommissionSelection() {
+    this.props.saveVideo(
+      _set('plutoData.projectId', null, this.props.video)
```

Comment:
we've changed the commission, so we need to erase the `projectId` and save the atom